tox.ini

```diff
@@ -11,6 +11,13 @@ envlist =
     py310
 isolated_build = True
 
+[gh-actions]
+python =
+    3.7: py37
+    3.8: py38
+    3.9: py39
+    3.10: py310
```

Comment:
Same question here about 3.11.

Reply:
Support for a new python version felt like it was deserving of it's own PR since there may be other code needed.